apps/address-service/domains/address/access/AddressSource.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generated by `createschema address.AddressSource 'source:Text;'`
+ */
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function canReadAddressSources ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    return false
+}
+
+async function canManageAddressSources ({ authentication: { item: user }, originalInput, operation, itemId }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return !!user.isAdmin
```

Comment:
It is already boolean type. So maybe it can be done like this:
```suggestion
    return user.isAdmin
```

Reply:
I'd like to return `boolean` despite of is `isAdmin` undefined or not.